public/app/dashboard/profileController.js

```diff
@@ -78,6 +78,8 @@
         analyticsService.track('Search for user profile', {
           'search_for_user': $scope.browse.searchedUserId
         });
+
+        crossToolRequest.scroll = null;
         $state.go('userprofile', {'userId': $scope.browse.searchedUserId});
```

Comment:
We can't use `loadProfileById(...)` in the line above because the `oi-select` (search input) would not be refreshed.